tests/testdata/configs/vector-config.yaml

```
@@ -0,0 +1,59 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: vector-config
+  namespace: cass-operator
+data:
+  vector.toml: |
+    [sources.systemlog]
+    type = "file"
+    include = [ "/var/log/cassandra/system.log" ]
+    read_from = "beginning"
+    fingerprint.strategy = "device_and_inode"
+
+    [sources.systemlog.multiline]
+    start_pattern = "^(INFO|WARN|ERROR|DEBUG|TRACE|FATAL)"
+    condition_pattern = "^(INFO|WARN|ERROR|DEBUG|TRACE|FATAL)"
+    mode = "halt_before"
+    timeout_ms = 10000
+
+    [transforms.parse_cassandra_log]
+    type = "remap"
+    inputs = [ "systemlog" ]
+    .del(source_type)
+    source = '''
+    . |= parse_groks!(.message, patterns: [
```

Nit: small typo here. I think the correct version is:
```suggestion
    source = '''
    del(.source_type)
    . |= parse_groks!(.message, patterns: [
```